### PR TITLE
Nodes group view

### LINKED ISSUE #1258: Edges can't be selected

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
A group of nodes can be represented in two different ways in the frontend:
- Pipeline/workbench like
- Settings&iFrame way

The first case was already covered in #1241 and what this PR brings is the second option.

In a single view we can display all the settings and iFrames of the internal nodes. Settings are stacked one after the other, while the iFrames are shown in a tab view manner.

Computational services:
TheOtherView1

Dynamic services:
TheOtherView2

## Related issue number
related to ITISFoundation/osparc-issues#18
closes #1258 

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: services/web/client/source/class/osparc/component/node/GroupNodeView.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    __addIFrame: function() {
+      this._iFrameLayout.removeAll();
+
+      const tabView = new qx.ui.tabview.TabView();
+      const innerNodes = this.getNode().getInnerNodes(true);
+      Object.values(innerNodes).forEach(innerNode => {
+        const iFrame = innerNode.getIFrame();
+        if (iFrame) {
+          const page = new qx.ui.tabview.Page().set({
+            layout: new qx.ui.layout.Canvas(),
```
Comment (ignapas) [inline services/web/client/source/class/osparc/component/node/GroupNodeView.js]:
```suggestion
            layout: new qx.ui.layout.Grow(),
```

File: services/web/client/source/class/osparc/component/node/GroupNodeView.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            showCloseButton: false
+          });
+          innerNode.bind("label", page, "label");
+          page.add(iFrame, {
```
Comment (ignapas) [inline services/web/client/source/class/osparc/component/node/GroupNodeView.js]: and just:
```javascript
page.add(iFrame);
```

File: services/web/client/source/class/osparc/component/node/GroupNodeView.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      this._addToMainView(this._settingsLayout);
+      this._addToMainView(this._mapperLayout, {
+        flex: 1
+      });
+    },
+
+    __addIFrame: function() {
+      this._iFrameLayout.removeAll();
+
+      const tabView = new qx.ui.tabview.TabView();
```
Comment (ignapas) [inline services/web/client/source/class/osparc/component/node/GroupNodeView.js]:
```suggestion
      const tabView = new qx.ui.tabview.TabView().set({
        contentPadding: 0
      });
```

**Comment by odeimaiz** [comment]
@ignapas Thanks for the review.

Since I would like to redesign the StudyEditor, I didn't put much effort in that logic. Anyway, all the requests were implemented.

**Comment by ignapas** [comment]
Buttons in ``ControlsBar`` change position when you press them (when you change the type of view) and also the height of the toolbar changes.

**Comment by odeimaiz** [comment]
> Buttons in `ControlsBar` change position when you press them (when you change the type of view) and also the height of the toolbar changes.

Since I would like to redesign the StudyEditor, I didn't put much effort in that logic. Anyway, all the requests were implemented.

**Pull request merged**